api/src/test/java/marquez/db/SearchDaoTest.java

```diff
@@ -0,0 +1,216 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package marquez.db;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.time.Instant;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import lombok.NonNull;
+import marquez.api.models.SearchFilter;
+import marquez.api.models.SearchResult;
+import marquez.api.models.SearchSort;
+import marquez.jdbi.MarquezJdbiExternalPostgresExtension;
+import org.glassfish.jersey.internal.guava.Lists;
+import org.jdbi.v3.core.Jdbi;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+/** The test suite for {@link SearchDao}. */
+@Tag("IntegrationTests")
```

Comment:
Why is this tagged as an integration test?

Reply:
Ahh, good point, I should be using the `DataAccessTests` tag. I'll open a PR to fix the tag.